mipengine/controller/node_landscape_aggregator.py

```diff
@@ -178,24 +177,23 @@ def _update(self):
                 )
 
                 (
-                    dataset_locations,
+                    dataset_location,
                     aggregated_datasets,
                 ) = _gather_all_dataset_info(datasets_per_node)
                 _update_data_models_with_aggregated_datasets(
                     data_models=compatible_data_models,
                     aggregated_datasets=aggregated_datasets,
                 )
 
-                dataset_locations = _get_dataset_locations_of_compatible_data_models(
-                    compatible_data_models, dataset_locations
+                dataset_location = _get_dataset_locations_of_compatible_data_models(
```

Comment:
I see you changed "locations" to "location", nevertheless the function returning the value still uses "locations", is this on purpose?